src/v2/openapi.yaml

```diff
@@ -0,0 +1,740 @@
+          type: array
+          items:
+            type: object
+            properties:
+              license1:
+                type: string
+                example: BSD License (BSD)
+              license2:
+                type: string
+                example: General Public License V3(GPL)
+        really_unknown:
+          type: array
+          items:
+            type: object
+            properties:
+              package:
+                type: string
+                example: django
+              license:
+                type: string
+                example: BSD License (BSD)
+      description: Data of licenses unknown to the system
+
+    ConflictPackages:
+      title: ConflictPackages
+      type: object
+      required: 
+        - package1
+        - package2
+        - license1
+        - license2
+      properties:
+        package1:
+          type: string
+          example: django
+        license1:
+          type: string
+          example: BSD License (BSD)
+        package2:
+          type: string
+          example: rustdoc
+        license2: 
+          type: string
+          example: General Public License V3(GPL)
+      description: Pair of packages which has conflict in license
+
+    LicenseAnalysis:
+      title: LicenseAnalysis
+      type: object
+      required: 
+        - outlier_packages
+        - conflict_packages
+        - current_stack_license
+        - unknown_licenses
+      properties:
+        outlier_packages:
+          type: array
+          items:
+            type: object
+            properties: {}
+        conflict_packages:
+          type: array
+          items:
+            $ref: '#/components/schemas/ConflictPackages'
+        current_stack_license:
+          type: object
+          properties: {}
+        unknown_licenses:
+          $ref: '#/components/schemas/UnknownLicenses'
+      description: License analyses data
+
+    GitHubDetails:
+      title: GitHubDetails
+      type: object
+      properties:
+        watchers:
+          type: integer
+          example: 1002
+        first_release_date:
+          type: string
+        total_releases:
+          type: integer
+          example: 12
+        issues:
+          type: object
+          properties: {}
+        pull_requests:
+          type: object
+          properties: {}
+        dependent_repos:
+          type: integer
+        open_issues_count:
+          type: integer
+        latest_release_duration:
+          type: string
+        forks_count:
+          type: integer
+          example: 123
+        contributors:
+          type: integer
+          example: 12
+        size:
+          type: string
+        stargazers_count:
+          type: integer
+          example: 50000
+        used_by:
+          type: array
+          items:
+            type: string
+        dependent_projects:
+          type: integer
+          example: 1534
+      description: Github details
+
+    PackageDetails:
```

Comment:
Can be included as properties within package

Reply:
Since `Package` model is used in Request body, it can't be removed.